.travis.yml

```diff
@@ -39,10 +39,19 @@ before_install:
 
 install:
   - ./continuous_integration/travis/install.sh;
-  - if [ "$USE_CONDA" = true ]; then source activate test-environment; fi
+  - if [ "$USE_CONDA" = true ]; then
+      source activate test-environment;
+    fi
+  - if [ "$USE_CONDA" = true ]; then
+      conda install pytest pytest-cov;
+    else
+      pip install pytest pytest-cov;
+    fi
+  - pip install pytest-qt
```

Comment:
I think it should be better to move all this instructions to install.sh. Maybe that helps to solve the problem.